config/rbac/role.yaml

```diff
@@ -2,67 +2,25 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  creationTimestamp: null
   name: manager-role
 rules:
-- apiGroups:
-  - apps
-  resources:
-  - deployments
-  verbs:
-  - '*'
-- apiGroups:
-  - apps
-  resources:
-  - deployments/finalizers
-  verbs:
-  - '*'
 - apiGroups:
   - ""
   resources:
   - configmaps
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
   - deployments
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
   - persistentvolumeclaims
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
   - persistentvolumes
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
   - pods
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
   - pods/exec
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
   - pods/log
   verbs:
   - '*'
 - apiGroups:
   - ""
   resources:
   - serviceaccounts
+  - services/finalizers
```

Comment:
What is generating this file ? It looks quite random what resources are handled together and when `*` is used or an explicit list of verbs. E.g services/finalizers and services have the same list of verbs, but defined in different sections, and one has a conflated list of verbs, the other uses a wildcard.

Should we manually curate this role definition file maybe ?

Reply:
Its generated from these annotations [here](https://github.com/llamastack/llama-stack-k8s-operator/blob/main/controllers/kubebuilder_rbac.go)